README.md

```diff
@@ -166,6 +167,42 @@ The following table describes keys and their values:
 | `:last_boot` | What caused the most recent boot. Whether this is reliable depends on the watchdog. |
 | `:heartbeat_timeout` | Erlang's heartbeat timeout setting. Note that the hardware watchdog timeout supersedes this since it reboots. |
 
+## Reboot and power off
+
+The `:heart.set_cmd/1` function can be used to trigger reboots and shutdowns.
+
+To understand why, it's good to review the alternatives. The other ways of
+rebooting with Nerves are either by exiting the Erlang VM via `:init.stop/0` or
+by running the `reboot(8)` or `poweroff(8)` shell commands. The latter two work
+by sending a signal to PID 1 (`erlinit`) that kills all OS processes and does
+the reboot and power off. These work well in practice, but have had failures
+blamed on them that can be further reduced by using Nerves Heart. For example,
+one failure was found where OS processes couldn't be started and therefore
+`reboot(8)` wasn't being called. This ended up causing a hang that wasn't
+resolved for a long time.  Since using Nerves Heart to reboot does not create OS
+processes, it would have this issue and if communication problems between Nerves
```

Comment:
```suggestion
processes, it would not have this issue and if communication problems between Nerves
```
I think you meant it wouldn't have this issue?